tools/apiview/parsers/apiview-treestyle-parser-schema/models.tsp

```diff
@@ -0,0 +1,150 @@
+model CodeFile {
```

Comment:
Add two scalars like this at the beginning of the file:
```typescript
scalar StructuredTokenId extends string;
scalar APITreeNodeId extends string;
```

now, you can use that to type better, for instance:
```typescript
model APITreeNode {

    Id: APITreeNodeId 
```

and the few properties that points to an id, would be more clear
```typescript
  /** Id to navigate to when StructuredToken is clicked. */
  NavigateToId?:  StructuredTokenId ;
```